components/ChangePasswordForm/ChangePasswordForm.js

```diff
@@ -56,58 +39,71 @@ export default class ChangePasswordForm extends React.Component {
       await onSuccess();
     } catch (error) {
       actions.setSubmitting(false);
-      this.setState({ errorMessage: getServerErrorMessage(error) });
+      setErrorMessage(getServerErrorMessage(error));
     }
   };
 
-  render() {
-    const { props, state } = this;
+  return (
+    <Formik
+      initialValues={initialValues}
+      onSubmit={handleSubmit}
+      validationSchema={passwordResetSubmitSchema}
+    >
+      {({ isSubmitting }) => (
+        <Form className={styles.PasswordResetSubmitForm}>
+          <div className={styles.row}>
+            <Field
+              type="password"
+              name="newPassword1"
+              label="Password*"
+              component={Input}
+              disabled={isSubmitting}
+              autoComplete="new-password"
+            />
+
+            <Field
+              type="password"
+              name="newPassword2"
+              label="Confirm Password*"
+              component={Input}
+              disabled={isSubmitting}
+              autoComplete="new-password"
+            />
+          </div>
+
+          <div className={styles.row}>
+            {errorMessage && <Alert type="error">{errorMessage}</Alert>}
+          </div>
 
-    return (
-      <Formik
-        initialValues={props.initialValues}
-        onSubmit={this.handleSubmit}
-        validationSchema={passwordResetSubmitSchema}
-      >
-        {({ isSubmitting }) => (
-          <Form className={styles.PasswordResetSubmitForm}>
-            <div className={styles.row}>
-              <Field
-                type="password"
-                name="newPassword1"
-                label="Password*"
-                component={Input}
-                disabled={isSubmitting}
-                autoComplete="new-password"
-              />
+          <div className={styles.row}>
+            <Button
+              className={styles.topMargin}
+              type="submit"
+              theme="secondary"
+              disabled={isSubmitting}
+            >
+              Submit
+            </Button>
+          </div>
+        </Form>
+      )}
+    </Formik>
+  );
+};
 
-              <Field
-                type="password"
-                name="newPassword2"
-                label="Confirm Password*"
-                component={Input}
-                disabled={isSubmitting}
-                autoComplete="new-password"
-              />
-            </div>
+ChangePasswordForm.propTypes = {
```

Comment:
Can you move these above the function definition?

Reply:
Below is the error I get when I move the prop types above function definition. Do you have any suggestions on a workaround this error???
``
ReferenceError: Cannot access 'SocialLoginGroup' before initialization
``

Reply:
You'll just need to convert the arrow functions to vanilla functions